assets/src/design-system/components/toggle/test/toggle.js

```diff
@@ -14,6 +14,22 @@
  * limitations under the License.
  */
 
-export const Z_INDEX = {
-  SNACKBAR: 15,
-};
```

Comment:
The diff is weird but this was its own file of z indexes. It is now deleted 😬 

The toggle is a new file.